units/systemd-nspawn@.service.in

```diff
@@ -36,6 +36,7 @@ TasksMax=16384
 DevicePolicy=closed
 DeviceAllow=/dev/net/tun rwm
 DeviceAllow=char-pts rw
+DeviceAllow=/dev/fuse rw
```

Comment:
After efedb6b0f3cff37950112fd37cb750c16d599bc7, now this also needs `m`, that is `/dev/fuse rwm`.